# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: ipc/glue/BackgroundChildImpl.cpp
# Commit: 8132dd939023
# Full Hash: 8132dd9390233328c8da7072f36859424a2b079d
# Author: Chris Martin <cmartin@mozilla.com>
# Date: 2020-10-14 21:42:48
# Regressor Bug: 1657404
# File Overlap Count: 2
# Description:
#   Bug 1657404 - Change PGamepadEventChannel to "refcounted protocol" r=handyman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D93119
# ==============================================================================

diff -r 186ca9e93e1d -r 8132dd939023 ipc/glue/BackgroundChildImpl.cpp
--- a/ipc/glue/BackgroundChildImpl.cpp	Wed Oct 07 11:34:02 2020 +0000
+++ b/ipc/glue/BackgroundChildImpl.cpp	Wed Oct 14 15:58:36 2020 +0000
@@ -605,20 +605,6 @@
   return true;
 }
 
-// Gamepad API Background IPC
-dom::PGamepadEventChannelChild*
-BackgroundChildImpl::AllocPGamepadEventChannelChild() {
-  MOZ_CRASH("PGamepadEventChannelChild actor should be manually constructed!");
-  return nullptr;
-}
-
-bool BackgroundChildImpl::DeallocPGamepadEventChannelChild(
-    PGamepadEventChannelChild* aActor) {
-  MOZ_ASSERT(aActor);
-  delete static_cast<dom::GamepadEventChannelChild*>(aActor);
-  return true;
-}
-
 dom::PGamepadTestChannelChild*
 BackgroundChildImpl::AllocPGamepadTestChannelChild() {
   MOZ_CRASH("PGamepadTestChannelChild actor should be manually constructed!");